common/src/main/java/org/geysermc/floodgate/AbstractLinkAccountCommand.java

```diff
@@ -0,0 +1,108 @@
+package org.geysermc.floodgate;
+
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import org.geysermc.floodgate.command.CommandMessage;
+import org.geysermc.floodgate.util.CommonMessage;
+import org.geysermc.floodgate.util.ICommandUtil;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Random;
+import java.util.UUID;
+
+@AllArgsConstructor
+public class AbstractLinkAccountCommand<PLAYER, COMMAND_UTIL extends ICommandUtil<PLAYER>> {
```

Comment:
Should be `public class AbstractLinkAccountCommand<P, C extends ICommandUtil<P>>` as using more than one capital letter in generics breaks the general naming conventions for them in Java.

Reply:
I wasn't aware of that, saw it used in some other projects too. Will change it